src/ProgressWindow.cpp

```diff
@@ -18,7 +18,7 @@ extern const char* kAppName;
 // TProgressView
 
 TProgressView::TProgressView(BRect frame, const char *name)
-	:	BView(frame, name, B_FOLLOW_LEFT | B_FOLLOW_TOP, B_PULSE_NEEDED)
+	:	BView(frame, name, B_FOLLOW_LEFT | B_FOLLOW_TOP, B_PULSE_NEEDED | B_SUPPORTS_LAYOUT)
```

Comment:
This should also use the layout-aware c'tor without the BRect and B_FOLLOW_* flags